src/filefinder.cpp

```diff
@@ -675,29 +687,52 @@ FileFinder::Directory FileFinder::GetDirectoryMembers(const std::string& path, F
 #  define readdir _wreaddir
 #elif _3DS
 	std::string wpath = path + "/";
+#elif PSP2
+#  define opendir sceIoDopen
+#  define closedir sceIoDclose
+#  define wpath path
+#  define dirent SceIoDirent
+#  define readdir sceIoDread
 #else
```

Comment:
Question: Why the need to define these and then later use the redefinitions?
Would make more sense to use them directly to me.